meeteval/wer/__main__.py

```diff
@@ -337,8 +375,111 @@ def positive_number(x: str):
 
         return x
 
-    def add_command(fn):
-        command_parser = commands.add_parser(
+    def add_argument(self, command_parser, name, p):
```

Comment:
Rename `add_argument` -> `add_arguments`?

Reply:
That function processes only one argument. That makes it easier in the future to add new arguments or disable arguments in the future in subclasses.